### PR TITLE
drafted blog post

### PR DESCRIPTION
**Pull request by arnaudvedy**

### CODE REVIEW

File: src/_stories/gis-update.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Dear readers  / followers, we have some exciting news!
+Our geographic data is now integrated with Poseidon, our data automation system.
+
+**What does this mean for you?**
+
+<!--more-->
+
+### Current geo data
+
+It means you now have access to the most current geographic data at all times. Some data sources are updated every month. For others, this process occurs on a daily basis. No matter the schedule of the dataset, our portal will now automatically update along with it. This ensures that as a user of the portal, you will be retrieving the most contemporary data available.
```
Comment (MrMaksimize) [inline src/_stories/gis-update.md]: contemporary data = sounds weird.  up-to-data data available = better

File: src/_stories/gis-update.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+![Data automation workflow for our zoning geo datasets](/assets/img/stories/zoning-workflow.jpg)
+
+### New formats
+
+Until recently, we were sharing geo data exclusively as shapefiles. This is one of the most commonly used format for geo data. Nevertheless, there are a few reasons why it should not be the only format of geo data offered on an open data portal:
+- It is not an open standard format.
+- It is actually multiple files. These files need to be zipped, making it inconvenient to share.
+- It is not a practical format for web developers. Few web mapping javascript libraries support shapefiles natively, which usually leads to data conversion.
+- It is not the most practical format for non-technical users. It requires software installation to be visualized / used for data analysis.
+For these reasons, we will be sharing our geo data in other formats from now on. The formats available for now are GeoJSON, TopoJSON, and Geobuf.
```
Comment (MrMaksimize) [inline src/_stories/gis-update.md]: We will be adding additional geodata formats.

Comment (MrMaksimize) [inline src/_stories/gis-update.md]: More clarification -> 

For these reasons, we will be additional geodata formats. For now (and we will be adding more) they are: GeoJSON, TopoJSON, and Geobuf.

File: src/_stories/gis-update.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+----------------------|-----------|---------| --------|-------------------
+right_of_way_datasd   |  Polygon  |138.1 MB | 4.7 MB  | ~ 29.4 x
+roads_datasd          | LineString|92.7 MB  | 7.9 MB  | ~ 11.7 x
+address_datasd        | Point     |175.3 MB | 15.7 MB | ~ 11.2 x -->
+
+![comparative table](/assets/img/stories/comparative-table-geobuf.png)
+
+So, you now have access to the most current geo data in multiple formats and we are pretty excited about it. What will you do with it?
+
+Let us know at [data@sandiego.gov](data@sandiego.gov) and stay tuned for more announcements!
```
Comment (MrMaksimize) [inline src/_stories/gis-update.md]: Wanna wait till we have the email or change it?

Comment (arnaudvedy) [inline src/_stories/gis-update.md]: I was actually thinking of using our twitter acct once we have one.

**Pull request merged**